<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MitbootWindow</class>
 <widget class="QMainWindow" name="MitbootWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MitbootWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="searchTextEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="searchButton">
        <property name="text">
         <string>Suchen</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="dbView"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="newButton">
        <property name="text">
         <string>Neuer Eintrag</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNeuer_Kontakt">
   <property name="text">
    <string>Neuer Kontakt</string>
   </property>
  </action>
  <action name="actionVerlassen">
   <property name="text">
    <string>Verlassen</string>
   </property>
  </action>
  <action name="actionVerwaltung_Postleitzahlen">
   <property name="text">
    <string>Verwaltung Postleitzahlen</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
